test/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -18,6 +18,9 @@ public class RepoDirectoriesProvider
         private string _corehostPackages;
         private string _corehostDummyPackages;
 
+        private string _targetRID;
+
+        public string TargetRID => _targetRID;
```

Comment:
Fair enough. I'd still suggest renaming the property to `TestTargetRID` to avoid confusion with the `TargetRID` BuildContext variable, which has the same value except that it uses "win7" for all non-ARM Windows platforms (see https://github.com/dotnet/core-setup/blob/e82cf3aef2537ba95aa5a0d258b1daf60d2a2dca/build_projects/dotnet-host-build/PrepareTargets.cs#L100-L103).